src/design-system/components/divider/Divider.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Typography, Divider } from '@mui/material'
+import { cn } from '~/utils/cn'
+
+import '~scss-components/divider/Divider.scss'
+
+const variants = ['fullWidth', 'inset', 'middle'] as const
+const orientation = ['vertical', 'horizontal'] as const
+const thickness = ['sm', 'md', 'lg'] as const
+const textAlign = ['left', 'right', 'center'] as const
+const type = ['ellipse', 'linear'] as const
+const size = ['small', 'large'] as const
+
+type DividerProps = {
+  variant: (typeof variants)[number]
+  orientation: (typeof orientation)[number]
+  thickness: (typeof thickness)[number]
+  textAlign: (typeof textAlign)[number]
+  caption: string
+  type: (typeof type)[number]
+  size?: (typeof size)[number]
+}
+
+const DividerComponent: React.FC<DividerProps> = ({
+  variant,
+  orientation,
+  thickness,
+  size,
+  caption,
+  textAlign,
+  type
+}) => {
+  const width = thickness === 'lg' ? '#9199A1' : '#C8CCD0'
```

Comment:
```suggestion
  const borderColor = thickness === 'lg' ? '#9199A1' : '#C8CCD0'
```
Rename it, please. It is not width

Reply:
fair enough, my bad